docs/make.jl

```diff
@@ -18,3 +18,5 @@ makedocs(;
         "API" => "api.md",
        ],
    )
+
+deploydocs(repo="github.com/dmlc/XGBoost.jl", devbranch="master")
```

Comment:
```suggestion
deploydocs(repo="github.com/dmlc/XGBoost.jl.git", devbranch="master")
```
Documenter.jl is very pedantic about the `.git`